README.md

```diff
@@ -23,8 +23,22 @@ class MyModel(models.Model):
     uuid = NativeShortUUIDField(unique=True, default=uuid.uuid4)
 ```
 
+If you want add the ability to search by shortuuid in `ModelAdmin` you need to inherit `NativeUUIDSearchMixin`
+```python
+from native_shortuuid.admin import NativeUUIDSearchMixin
+
+@admin.register(models.MyModel)
+class MyModelAdmin(NativeUUIDSearchMixin, admin.ModelAdmin):
+    search_uuid_fields = ('uuid', )
```

Comment:
Let us remove this attribute from here as it is not part of the default behavior, and update ADMIN_AUTO_EXTRACT_UUID_SEARCH_FIELDS setting part below to tell user that they need to set this array if they turn off this setting for some reason.